src/components/home/LineSeparator.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import { styled } from '@material-ui/styles';
+
+import { ThickLine, spacing } from '../styledComponents';
+
+const Container = styled(Grid)({ margin: `${spacing(12)} 0px` });
+
+const LineContainer = styled(Grid)({
+  position: 'absolute',
+  right: '0px',
+});
+
+const LineSeparator = () => (
```

Comment:
Is this going to be a standardized component that we'll re-use more places than just the homepage? If so, it's probably worth creating a little "UI" library not in `/home` so it's easy to reuse